vehicle/mkvi/software/charging/charger.c

```diff
@@ -115,13 +113,13 @@ int main(void) {
             }
         }
         //safety checks
-        if (charging_fbk.hardware_fault) {
+        if (hardware_fault) {
             charging_cmd.enable = false;
         }
-        if (charging_fbk.temperature_protection) {
+        if (temperature_protection) {
             charging_cmd.enable = false;
         }
-        if (charging_fbk.charging_voltage > charging_cmd.max_voltage || charging_fbk.charging_current > charging_cmd.max_current) {
+        if (charging_voltage > charging_cmd.max_voltage || charging_current > charging_cmd.max_current) {
             charging_cmd.enable = false;
         }
 
```

Comment:
We never actually send out the message - perhaps there should be a spi_send() command somewhere in the superloop?